app/components/Form/FormComponentProps.tsx

```diff
@@ -1,6 +1,8 @@
-import type { AjvError } from "@rjsf/core";
+import type { AjvError, ObjectFieldTemplateProps } from "@rjsf/core";
 export default interface FormComponentProps {
```

Comment:
rjsf enforces `schema` to be there, so we can't use the `FormProps` as a base...this small interface is just here to group what our different forms might need since they implement rjsf in turn